recipes/leftbar/contact_row/contact_row_variants.story.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="d-w464">
+    <dt-stack gap="600">
+      <div>
+        <h3>
+          With user status
+        </h3>
+        <dt-recipe-contact-row
+          name="Jaqueline Nackos"
+          :avatar-src="defaultImage"
+          avatar-presence="active"
+          user-status="Good Morning! :smile:"
+        />
+      </div>
+      <div>
+        <h3>
+          with away presence
+        </h3>
+        <dt-recipe-contact-row
+          name="Jaqueline Nackos"
+          avatar-presence="away"
+          presence-text="Away"
+          user-status="Out for a bit"
+          :avatar-src="defaultImage"
+        />
+      </div>
+      <div>
+        <h3>
+          with busy presence
+        </h3>
+        <dt-recipe-contact-row
+          name="Jaqueline Nackos"
+          avatar-presence="busy"
+          presence-text="In a meeting"
+          user-status="Meetings all day"
+          :avatar-src="defaultImage"
+        />
+      </div>
+      <div>
+        <h3>
+          With unread count
+        </h3>
+        <dt-recipe-contact-row
+          name="Jaqueline Nackos"
+          :avatar-src="defaultImage"
+          avatar-presence="active"
+          unread-count="5"
+          :has-unreads="true"
+        />
+      </div>
+      <div>
+        <h3>
+          Selected
+        </h3>
+        <dt-recipe-contact-row
+          name="Jaqueline Nackos"
+          :avatar-src="defaultImage"
+          avatar-presence="active"
+          :selected="true"
+        />
+      </div>
+      <div>
+        <h3>
+          With no image
+        </h3>
+        <dt-recipe-contact-row
+          name="Jaqueline Nackos"
+          avatar-presence="active"
+        />
+      </div>
+    </dt-stack>
+  </div>
+</template>
+
+<script>
+import DtRecipeContactRow from './contact_row';
+import { DtStack } from '@/components/stack';
+const defaultImage = require('./../../../components/avatar/person.png');
```

Comment:
const defaultImage = require('@/components/avatar/person.png');

Reply:
I can see this before the image loads correctly
<img width="592" alt="image" src="https://user-images.githubusercontent.com/89984179/222817180-1982991e-3ee2-476a-85eb-93c125587bca.png">

but on the pr preview its seems to load the fallback image, cool

<img width="592" alt="image" src="https://user-images.githubusercontent.com/89984179/222819349-d39a6c77-0f6b-4c71-86e6-600bbdc11c88.png">


At the end its works correctly on both, you can test it with incognito mode and slow 3g on network tab

Reply:
Eek, I have zero interest right now in viewing the DP app with incognito mode and slow 3g.

Reply:
hmm yeah seems like a local thing